styles.css

```diff
@@ -0,0 +1,165 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+/* body */
+body {
+    font-family: AvenirNext,Avenir,"Helvetica Neue",Helvetica,Arial,sans-serif;
+    background-color: #ECEEF1;
+}
+
+/* header */
+header {
+    display: flex;
+}
+.nav-text{
+    margin-top: 20px;
+    margin-right: 6px;
+    font-size: 12px;
+    margin-left: 1050px;
+}
+
+.nav-btn {
+    margin-top: 20px;
+    background: #0077C5;
+    border-radius: 4px;
+    margin-left: 10px;
+    border: 1px solid #4c4c4c;
+    color: #FFF;
+    width: 100px;
+    height: 35px;
+}
+
+/* middle section */
+.logo img{
+    width: 100px;
+    margin-left: 600px;
+}
+.image-logo{
+    width: 400px;
+    margin-left: 500px;
+    margin-top: 30px;
+    margin-bottom: -130px;
+}
```

Comment:
You can use flex properties to make the logos with their text aligned with the proper spacing. Since you didn't take care of that

Reply:
And that's It. When finished! kindly re-submit your Pull request for another code review. :muscle: :muscle: